### DISCUSSION TITLE
'Releases' showing 2 users and 2 session on only 1 app start.

### DISCUSSION THREAD

**Discussion by steffos** [getsentry/sentry-dotnet#3448] [Q&A]
How is it possible that I, on the 'Releases' page get:
* 2 sessions
* 2 users
when I only have started the app once? I start it and let it run and wait for a minute or two.

I start the app with only the o.Dsn option set and 
log one error with SentrySdk.CaptureException(ex)

that's it.

When clicking the 'User Count' row on the 'Releases' page I get a graph telling me that 
* 1 Healthy 
* 1 Errored 
user count has been registered.

I'm 100% certain that the app has only been started once.

What could be the reason for this behaviour?

**Comment by bitsandfoxes** [comment]
Hey @steffos, what kind of application is it that you're seeing this happen in? I wonder if this is related to one of the native SDKs doing its own thing a second time.

**Comment by steffos** [comment]
This has been identified as a bug - https://github.com/getsentry/sentry-dotnet/issues/3452
